legacy/scripts/exec-generate-insights-configmap.sh

```diff
@@ -55,7 +55,9 @@ echo "Running sbom scan using trivy"
 echo "Image being scanned: ${IMAGE_FULL}"
 echo "Using image for scan ${IMAGECACHE_REGISTRY}${INSIGHTS_SCAN_IMAGE}"
 
-DOCKER_HOST=docker-host.lagoon.svc docker run --rm -v /var/run/docker.sock:/var/run/docker.sock ${IMAGECACHE_REGISTRY}${INSIGHTS_SCAN_IMAGE} image --skip-java-db-update ${IMAGE_FULL} --format ${SBOM_OUTPUT} | gzip > ${SBOM_OUTPUT_FILE}
+# Setting JAVAOPT to skip the java db update, as the upstream image comes with a pre-populated database
+JAVAOPT="--skip-java-db-update"
+DOCKER_HOST=docker-host.lagoon.svc docker run --rm -v /var/run/docker.sock:/var/run/docker.sock ${IMAGECACHE_REGISTRY}${INSIGHTS_SCAN_IMAGE} image ${JAVAOPT} ${IMAGE_FULL} --format ${SBOM_OUTPUT} | gzip > ${SBOM_OUTPUT_FILE}
```

Comment:
No need to specify the `DOCKER_HOST` here, as it is already defined as an envvar here https://github.com/uselagoon/build-deploy-tool/blob/main/Dockerfile#L107

And once we move to https://github.com/uselagoon/remote-controller/pull/291, it will be set by the remote-controller.

Reply:
okay cool - I'll remove it :)